grails-app/controllers/org/pih/warehouse/user/DashboardController.groovy

```diff
@@ -78,10 +78,10 @@ class DashboardController {
 
         def shipment = Shipment.findByShipmentNumber(params.searchTerms)
         if (shipment) {
-            if (shipment?.isStockMovement()) {
+            if (shipment?.requisition) {
```

Comment:
I think we should remove this condition and have one redirect to stock movement's details page.